k8s/operators/src/pool/context.rs

```diff
@@ -238,6 +239,24 @@ impl ResourceContext {
         Ok(Action::requeue(Duration::from_secs(30)))
     }
 
+    /// Removes pool expand annotation from the CR.
+    async fn patch_metadata(&self) -> Result<(), Error> {
+        let patch = Patch::Merge(json!({
+            "metadata": {
+                "annotations": {
+                    "openebs.io/expand": null
+                }
+            }
+        }));
+        let ps = PatchParams::apply(WHO_AM_I);
```

Comment:
Apply doesn't seem correct here, we're not doing server side apply, we're doing a merge